kernel/packages/decentraland-ecs/src/decentraland/Events.ts

```diff
@@ -74,6 +74,12 @@ export const onLeaveSceneObservable = new Observable<IEvents['onLeaveScene']>(cr
 /* @deprecated Use onLeaveSceneObservable instead. */
 export const onLeaveScene = onLeaveSceneObservable
 
+/**
+ * This event is triggered once the scene should start.
```

Comment:
Can we have a more narrow description? Isn't the scene supposed to have started to at the stage this `const` is being evaluated? It may be confusing.

It is not clear to me what is this event actually signaling.

Reply:
This happens when the Scene is ready and it loaded its models.

Maybe change the event name to:

`onSceneReadyObservable`?

One example in the world for thid event is when a scene is loading we see green limiters.  When those limiters disappear (the scene finish loading) it would execute this event.

Reply:
Ohh I see, "onSceneReadyObservable" sounds good enough for the use case, maybe something like "Event triggered after all the resources of the scene were loaded (models, textures, etc...)" may help the description